userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -594,6 +594,10 @@ contents:
 ```go-html-template
```

Comment:
Should we add something in this section to the effect that you can only have shortcodes inside your includes from x version of Hugo, otherwise users who haven't updated might be disappointed to see shortcodes not rendering?

Reply:
Good point. Added a note to the user guide.

Thanks!